README.md

```diff
@@ -130,39 +148,38 @@ curl 'http://127.0.0.1:7700/indexes/movies/search?q=botman+robin&limit=2' | jq
 }
 ```
 
-#### With the Web Interface
+#### Use the Web Interface
 
-MeiliSearch provides a simple web interface containing a search bar in order to quickly test the instant search experience with a given set of documents.
+We also deliver an **out-of-the-box web interface** in which you can test MeiliSearch interactively.
 
-This web interface is available in your browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700).
+You can access the web interface in your web browser at the root of the server. The default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700). All you need to do is open your web browser and enter MeiliSearch’s address to visit it. This will lead you to a web page with a search bar that will allow you to search in the selected index.
 
 ### Documentation
 
-Now, that you have a running MeiliSearch, you can learn more and tune your search engine using [the documentation](https://docs.meilisearch.com).
+Now that your MeiliSearch server is up and running, you can learn more about how to tune your search engine in [the documentation](https://docs.meilisearch.com).
 
 ## How it works
 
-MeiliSearch uses [LMDB](https://en.wikipedia.org/wiki/Lightning_Memory-Mapped_Database) as the internal key-value store. The key-value store allows us to handle updates and queries with small memory and CPU overheads. The whole ranking system is [data oriented](https://github.com/meilisearch/MeiliSearch/issues/82) and provides great performances.
+MeiliSearch uses [LMDB](https://en.wikipedia.org/wiki/Lightning_Memory-Mapped_Database) as the internal key-value store. The key-value store allows to handle updates and queries with small memory and CPU overheads. The whole ranking system is [data oriented](https://github.com/meilisearch/MeiliSearch/issues/82) and ensures great performances.
 
-You can [read the deep dive](deep-dive.md) if you want more information on the engine; it describes the whole process of generating updates and handling queries. Also, you can take a look at the [typos and ranking rules](typos-ranking-rules.md) if you want to know the default rules used to sort the documents.
+You can read [this document](deep-dive.md) if you want to dive deeper into the engine. The whole process of generating updates and handling queries is described in it. Besides, to learn the default rules used for sorting documents, you can take a look at this [typos and ranking rules explanation](typos-ranking-rules.md).
```

Comment:
I would prefer we remove this paragraph as it is really outdated, I will write a better deep dive later on ;)

Reply:
Should I remove from: `MeiliSearch uses` to `explanation](typos-ranking-rules.md).`?